### PR TITLE
Mark initial set of HTTP semantic conventions as frozen

### PR DESCRIPTION
**Pull request by trask**
Fixes https://github.com/open-telemetry/opentelemetry-specification/issues/3279

### CODE REVIEW

File: specification/metrics/semantic_conventions/http-metrics.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ operations. By adding HTTP attributes to metric events it allows for finely tune
 
 ### Metric: `http.server.duration`
 
+**Status**: [Experimental, Feature-freeze][DocumentStatus]
```
Comment (joaopgrassi) [inline specification/metrics/semantic_conventions/http-metrics.md:57]: It's unrelated to this PR, but I wonder if we shouldn't adapt our semconv tool to also generate this. We have the field for it in the [YAML schema ](https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py#L38). Except we don't have "feature-freeze" but I guess we could add that.

**Comment by reyang** [comment]
Hopefully this is the last PR for https://github.com/orgs/open-telemetry/projects/41/views/4.

image

**Review by reyang — APPROVED** [review]

**Review by mateuszrzeszutek — APPROVED** [review]

File: specification/metrics/semantic_conventions/http-metrics.md (hunk 1 above)
Comment (arminru) [reply specification/metrics/semantic_conventions/http-metrics.md:58]: @lmolkova proposed this in https://github.com/open-telemetry/build-tools/pull/170 but this is pending on https://github.com/open-telemetry/opentelemetry-specification/issues/3459 I think.

Comment (arminru) [reply specification/metrics/semantic_conventions/http-metrics.md:58]: @lmolkova proposed this in https://github.com/open-telemetry/build-tools/pull/170 but this is pending on https://github.com/open-telemetry/opentelemetry-specification/issues/3459 I think.

File: specification/metrics/semantic_conventions/http-metrics.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ linkTitle: HTTP
 
 # Semantic Conventions for HTTP Metrics
 
-**Status**: [Experimental][DocumentStatus]
+**Status**: [Experimental, partial feature-freeze][DocumentStatus]
```
Comment (jsuereth) [inline specification/metrics/semantic_conventions/http-metrics.md:7]: Should this be "Mixed" then?

**Review by jsuereth — APPROVED** [review]

File: specification/metrics/semantic_conventions/http-metrics.md (hunk 2 above)
Comment (reyang) [reply specification/metrics/semantic_conventions/http-metrics.md:7]: Nope. I think "Mixed" means the doc has a mixture of [Lifecycle status](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/document-status.md#lifecycle-status) (Experimental/Stable/Deprecated), while we only have "Experimental" here.

[Feature-freeze](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/document-status.md#feature-freeze) is orthogonal to Lifecycle status.

**Comment by pellared** [comment]
I kindly suggest taking a look at https://github.com/open-telemetry/semantic-conventions/issues/128 before going Stable.

When I read [feature-freeze definition](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md#feature-freeze) I feel that it may be too late.

Personally, I do not feel comfortable that so many attributes are marked as `Required` and their values may contain PII. Take notice that it may not be easy to use a Collector to get rid of the attributes e.g. "mobile" instrumentation (Android, iOS).

**Comment by reyang** [comment]
> When I read [feature-freeze definition](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md#feature-freeze) I feel that it may be too late. Hopefully, I am wrong.

https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/document-status.md#feature-freeze

_Changes that address production issues with existing features are still accepted._

**Comment by pellared** [comment]
> Changes that address production issues with existing features are still accepted.

I just want to share that the example of "mobile instrumentation" and PII is a real problem for one of our customers 😉 

I do not have capacity to propose any PR in the following days (maybe even weeks). But I proposed some ideas in https://github.com/open-telemetry/semantic-conventions/issues/128

**Comment by Oberon00** [comment]
> Changes that address production issues with existing features are still accepted.

I don't think this is meant for _new_ production issues and critical things. Otherwise a missing feature could also cause production issues.

**Comment by Oberon00** [comment]
If we want to address PII things, we need to lift the feature freeze (I think this will happen after stabilization is complete & released?)

File: specification/metrics/semantic_conventions/http-metrics.md [hunk 3]
```diff
@@ -55,6 +55,8 @@ operations. By adding HTTP attributes to metric events it allows for finely tune
 
 ### Metric: `http.server.duration`
 
+**Status**: [Experimental, Feature-freeze][DocumentStatus]
```
Comment (jack-berg) [inline specification/metrics/semantic_conventions/http-metrics.md:58]: @trask what does it mean for a metric to have both experimental and Feature freeze simultaneously?

Comment (reyang) [reply specification/metrics/semantic_conventions/http-metrics.md:58]: Feature-freeze is orthogonal to Experimental/Stable https://github.com/open-telemetry/opentelemetry-specification/blob/ad1537a46eb856eeb915786fd7cf81e07bdeb426/specification/document-status.md?plain=1#L19.

**Pull request merged**